README.md

```diff
@@ -259,7 +259,65 @@ BundleKeys.endpoints := Map("akka-remote" -> Endpoint("tcp"))
 
 In the above, no declaration of `services` is required as akka remoting is an internal, cluster-wide TCP service.
 
-## play[23|24|25]-conductr-bundle-lib
+## play25-conductr-bundle-lib
```

Comment:
In Play 2.5 we now have a Java and Scala API. Therefore I'd make two section under `play25-conductr-bundle-lib`:
- Scala
- Java

Each sub section should contain only the relevant Scala / Java documentations. As a result we would duplicate some text but as a user it is much easier to read.